package.json

```diff
@@ -41,7 +41,7 @@
   },
   "resolutions": {
     "**/ansi-regex": "^5.0.1",
-    "**/log4js": "6.3.0",
+    "**/log4js": "6.4.6",
```

Comment:
Imperative is being updated to use log4js 6.4.6 as its dependency 9and the CLI will be updated shortly after)
Not sure if we want to keep the resolution here or not. 

I don't necessarily have a strong preference other than the resolutions list getting unnecessarily long.

Reply:
@zFernand0 when was the cli/imperative scheduled to have this update in a release? I know this was mentioned 5 days ago, is that release out yet?

Reply:
It is included in Imperative 5.2.2, CLI 7.1.3
Those versions should likely make their way to public NPM tomorrow 😋 
We can open another PR to include them throughout the week 😋